### PR TITLE
Snap package instructions

### PR DESCRIPTION
**Pull request by amzn-changml**
## Change summary

Adds a new section to the Linux setup with install instructions for Snap packages. There are additional modifications for separating the deb and snap sections.

### Submission Checklist:

* [X] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [X] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [X] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [X] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 1]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+{{< /note >}}
+
+1. Download the snap package from the [O3DE download](https://o3de.org/download/#linux) page or the [Snap store](https://snapcraft.io/o3de)
+
+1. Depending on disto, you will need to install `snapd` in order to install the snap package. [Refer to this guide for instructions specific to your distro](https://snapcraft.io/docs/installing-snapd)
+
+1. If the snap package has been downloaded from the O3DE page, use this command to install the package
+   ```shell
+   snap install --classic --dangerous <o3de snap package filename>.snap
+   ```
```
Comment (AMZN-alexpete) [inline content/docs/welcome-guide/setup/installing-linux.md:122]:
```suggestion
   ```shell
   snap install --classic --dangerous <o3de snap package filename>.snap
   ```
   The [`--dangerous` ](https://snapcraft.io/docs/install-modes#heading--dangerous)option is not needed if installed from the snap store.
```

**Review by AMZN-alexpete — APPROVED** [review]
looks good just a couple suggestions about explaining what the snap flags mean

File: content/docs/welcome-guide/setup/installing-linux.md (hunk 1 above)
Comment (amzn-changml) [reply content/docs/welcome-guide/setup/installing-linux.md:122]: This instruction is if the snap package is downloaded form o3debinaries.org, in the event that the snap store approval is not ready by release. It's needed as there will not be any signed hash metadata from the snap store if the package is not published.

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 2]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
     (Reading database ... 66411 files and directories currently installed.)
     Removing o3de (0.0.0.0) ...
     ```
+
+## Installing O3DE for Linux from a Snap package
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:109]: For consistency:
```suggestion
## Installing O3DE from a Snap package
```

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 3]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+
+{{< note >}}
+The Snap package is experimental and may run into issues on some distros. This has been tested on Ubuntu 20.04 and 22.04 LTS
+{{< /note >}}
+
+1. Download the snap package from the [O3DE download](https://o3de.org/download/#linux) page or the [Snap store](https://snapcraft.io/o3de)
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:115]:
```suggestion
1. Download the Snap package from the [O3DE download](https://o3de.org/download/#linux) page or the [Snap store](https://snapcraft.io/o3de)
```

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 4]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+
+1. Depending on disto, you will need to install `snapd` in order to install the snap package. [Refer to this guide for instructions specific to your distro](https://snapcraft.io/docs/installing-snapd)
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:117]:
```suggestion
1. Depending on the distro, you will need to install `snapd` in order to install the Snap package. [Refer to this guide for instructions specific to your distro](https://snapcraft.io/docs/installing-snapd).
```

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 5]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+
+1. If the snap package has been downloaded from the O3DE page, use this command to install the package
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:119]:
```suggestion
1. If you downloaded the Snap package from the O3DE download page, use the following command to install the package:
```

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 6]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+   ```
+   The following output will be displayed if successful:
+   ```shell
+   o3de <version> installed
+   ```
+   Where `<version>` is the version of the installer. Example: `23.05.0`
+
+O3DE will be installed in the default location: `/snap/o3de/x1/<version>`, where `<version>` is the version of the installer. Example: `23.05.0`.
+
+When installation is complete, you can find **Project Manager** and other tools in `<install-directory>/bin/Linux/profile/Default`. `snapd` will add this to your shell environment's path.
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:131]:
```suggestion
When installation is complete, you can find **Project Manager** and other tools in `<install-directory>/bin/Linux/profile/Default`. `snapd` will add the `Default` directory to your shell environment's path.
```

**Review by willihay — CHANGES_REQUESTED** [review]

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 7]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
     (Reading database ... 66411 files and directories currently installed.)
     Removing o3de (0.0.0.0) ...
     ```
+
+## Installing O3DE for Linux from a Snap package
+
+{{< note >}}
+The Snap package is experimental and may run into issues on some distros. This has been tested on Ubuntu 20.04 and 22.04 LTS
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:112]:
```suggestion
The Snap package is experimental and may run into issues on some distros. This has been tested on Ubuntu 20.04 and 22.04 LTS.
```

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 8]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+
+1. If the snap package has been downloaded from the O3DE page, use this command to install the package
+   ```shell
+   snap install --classic --dangerous <o3de snap package filename>.snap
+   ```
+   The following output will be displayed if successful:
+   ```shell
+   o3de <version> installed
+   ```
+   Where `<version>` is the version of the installer. Example: `23.05.0`
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:127]:
```suggestion
   Where `<version>` is the version of the installer. Example: `23.05.0`.
```

File: content/docs/welcome-guide/setup/installing-linux.md [hunk 9]
```diff
@@ -105,3 +105,43 @@ Example of launching Project Manager from the shell:
...
+When installation is complete, you can find **Project Manager** and other tools in `<install-directory>/bin/Linux/profile/Default`. `snapd` will add this to your shell environment's path.
+
+Example of launching Project Manager from the shell:
+```shell
+o3de
+```
+
+## Removing O3DE installed from a Snap package
+
+1. Run `snap` to remove o3de:
```
Comment (willihay) [inline content/docs/welcome-guide/setup/installing-linux.md:140]:
```suggestion
1. Run `snap` to remove O3DE:
```

File: content/docs/welcome-guide/setup/installing-linux.md (hunk 1 above)
Comment (amzn-changml) [reply content/docs/welcome-guide/setup/installing-linux.md:122]: That said, will remove this flag if we do get the snap published

**Review by willihay — APPROVED** [review]

**Review by phuchau1989 — APPROVED** [review]

**Pull request merged**